src/content/docs/concepts/programs.mdx

```diff
@@ -21,7 +21,7 @@ In Membrane, you build custom functionality (bots, workflows, etc.) by writing p
 
 As a starting user, you'll first install [drivers](/concepts/drivers) to talk to APIs for apps you already use, then write programs using the nodes exposed by those drivers to build something useful to you or your team. You can also always use `fetch` and/or build your own drivers if you want.
 
-For example, you could write a program that listens to a GitHub repo and sends you messages in Discord when there are new commits (this program [already exists](https://github.com/membrane-io/example-github-discord-tracker/tree/901e198ae309294024d6a0eee4fd37b67b865b1a), if you're interested).
+For example, you could write a program that listens to a GitHub repo and sends you messages in Discord when there are new commits (this program <a href="https://github.com/membrane-io/example-github-discord-tracker/tree/901e198ae309294024d6a0eee4fd37b67b865b1a" target="_blank">already exists</a>, if you're interested).
```

Comment:
Hm yep [that is possible in MDX/Astro](https://docs.astro.build/en/guides/markdown-content/#assigning-custom-components-to-html-elements), but as far as I can tell it would require importing the custom link and exporting `components` in each mdx file, so it'd be just as much boilerplate and more indirection. Plus, most of our files/pages are plain `.md` right now. 

Other libraries (like `mdx-remote` that I use on my personal site) [make it easier](https://github.com/pmillspaugh/petemillspaugh.com/blob/main/src/components/Post/Post.tsx#L61).